subcommittees/mentoring/PracticumProposal.md

```diff
@@ -0,0 +1,192 @@
+# Instructor Training Practicum Proposal
+
+## Background and motivation
+
+There are several reasons to offer a follow-up training experience to the base instructor training.
+These reasons are: 
+
+First, many freshly minted instructors have stated that once they are out of teacher training, they
+still feel that they are not comfortable teaching our lessons. One of the things that can be done to
+alleviate this is to ensure that they have worked through the lesson material that they will be 
+teaching from, and also had experience in teaching from the material. Through this, we ensure that
+they know the lesson material, and have helped and guided them through what they will be teaching.
+It is likely that this will lower the threshold to teaching their first workshop.
+
+Second, we are expecting that the people we admit to be our teachers already know certain things.
+This might not be the case - we could have people who are great at unix but who know little about
+git, and people with R knowledge who would not be able to manoeuvre on the command line. Thus, there
+might be gaps in their knowledge, which the students themselves may or may not be aware of. By having
```

Comment:
I _thought_ I had managed to use consistent vocabulary (i.e. the list above), but turned out I had a few "students" left. Now eliminated.